canonical/canonical.com
# WD-12223 Refreshed /data/opensearch

## Done

- Refreshed /data/opensearch
- This is part of many pages that needs to be created under /data/opensearch , hence some of navigation links might return 404.
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- [Demo](https://canonical-com-1308.demos.haus/data/opensearch)
- [Copydoc](https://docs.google.com/document/d/1nVdoKoPrXw8lsvHEtMichlPNfxYU9Uj1zmKowcUXQ_w/edit)
- [Figma file](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=10-10158&t=hD5i7YosKZ9Ru1dD-0)

## Issue / Card

Fixes # [WD-12223](https://warthogs.atlassian.net/browse/WD-12223)




[WD-12223]: https://warthogs.atlassian.net/browse/WD-12223?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
